openfermioncirq/optimization/black_box.py

```diff
@@ -46,8 +46,21 @@ class BlackBox(metaclass=abc.ABCMeta):
             a list of tuples of the form (low, high), where low and high are
             lower and upper bounds on a parameter. The number of tuples
             should be equal to the dimension of the black box.
+        cost_of_evaluate: If specified, then calls to `evaluate` will be
+            redirected to `evaluate_with_cost` with the specified cost.
     """
 
+    def __init__(self,
+                 cost_of_evaluate: Optional[float]=None,
+                 **kwargs) -> None:
+        """
+        Args:
+            cost_of_evaluate: An optional cost associated with the
```

Comment:
I still feel this cost_of_evaluate business is not well explained. What does this float mean!?